version/version.go

```diff
@@ -124,7 +125,7 @@ func (o *osRelease) DisplayVersion() string {
 // readOsRelease reads and parses the os-release file.
 func readOsRelease() (*osRelease, error) {
 	for _, path := range []string{"/etc/os-release", "/usr/lib/os-release"} {
-		f, err := os.Open(path)
+		f, err := os.Open(filepath.Clean(path))
```

Comment:
Same as above, but even more obvious. What should be cleaned here? Those strings are hard-coded absolute paths. I would further ignore this linter warning as adding `filepath.Clean` just obscures things.

Reply:
Same here as well!